md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/TableColumnLineageAPI.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.wipro.ats.bdre.md.rest;
+
+
+import com.wipro.ats.bdre.ldg.GetDotForTable;
+import com.wipro.ats.bdre.md.api.base.MetadataAPIBase;
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import java.security.Principal;
+
+
+/**
+ * Created by arijit on 1/9/15.
+ */
+@Controller
+@RequestMapping("/tabcollineage")
+public class TableColumnLineageAPI extends MetadataAPIBase {
+    private static final Logger LOGGER = Logger.getLogger(TableColumnLineageAPI.class);
+
+    /**
+     * This method is used to see the relationship between processes linked to a particular table name and/or column name.
+     * It also generates the dot string for visualisation of the links.
+     *
+     * @param tableName, colName
+     * @return restWrapper It contains an instance of LineageInfo.
+     */
+    @RequestMapping(value = {"", "/"}, method = RequestMethod.GET)
+    public
+    @ResponseBody
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)